src/components/main-nav/main-nav.js

```diff
@@ -6,5 +6,50 @@ export default class MainNav {
   constructor (node) {
     this.node = document.querySelector(node)
     this.node.innerHTML = template(mainNavData)
+    this.items = document.querySelectorAll('.menu__item')
+    this.links = document.querySelectorAll('.menu__link')
+    this.deploySubmenuEvent()
+    this.deploySubmenuEventLarge()
+    this.toggleMenuEvent()
+  }
+
+  deploySubmenuEvent () {
+    this.items.forEach((_, index, array) => {
+      array[index].addEventListener('click', e => {
+        this.resetClass(index)
+        array[index].querySelector('.submenu').classList.toggle('show')
+        this.links[index].classList.toggle('animated')
+      })
+    })
+  }
+
+  deploySubmenuEventLarge () {
+    if (window.screen.width >= 1024) {
```

Comment:
You can create an utility that can provide a boolean to know in which viewport the user is. It could look like: `viewport.is('large')` or `viewport.isLarge()` depending on how do you want to call it. Numbers as `1024` should be also written as a constant somewhere.